src/elementary_operators.jl

```diff
@@ -230,6 +241,23 @@ function operator(fx::ITensorNetworkFunction)
   return operator
 end
 
+"Create a product state of a given bit configuration"
+function delta_xyz(s::IndsNetworkMap, xs::Vector, dimensions::Vector{Int}; kwargs...)
+  ind_to_ind_value_map = calculate_ind_values(s, xs, dimensions)
+  ps = [string(ind_to_ind_value_map[s[v][1]]) for v in vertices(s)]
+  return ttn(ps, indsnetwork(s); kwargs...)
+end
+
+function delta_xyz(s::IndsNetworkMap, xs::Vector; kwargs...)
+  return delta_xyz(s, xs, [i for i in 1:length(xs)]; kwargs...)
+end
+
+"Create a product state of a given bit configuration of a 1D function"
+function delta_x(s::IndsNetworkMap, x::Number, kwargs...)
```

Comment:
So this function returns the `ITensorNetwork` representing `f(x') = 1 if x' = x and 0 otherwise` right? So a normalized version of the Dirac delta function.

This is useful but surely we should move all of this to the `elementary_functions.jl` code as these are function constructors. Also we should construct these as `ITensorNetworks`, I don't think there is any need for it to be limited to a tree? This should be doable via:

```
tn = ITensorNetwork(v -> string(ind_to_ind_value_map[only(s[v])], indsnetwork(s)) 
return ITensorNetworkFunction(tn, s)
```